### ISSUE TITLE
Fix `Panel` rendering when it 2 `Panel`s are used

### ISSUE THREAD

**Issue by zhiburt** [zhiburt/tabled#50]
This is an example

```rust
use tabled::{Footer, Header, MaxWidth, Modify, Row, Style, Cell};

fn main() {
    let message = r#"The terms "the ocean" or "the sea" used without specification refer to the interconnected body of salt water covering the majority of the Earth's surface"#;
    let link = r#"https://en.wikipedia.org/wiki/Ocean"#;

    let oceans = ["Atlantic", "Pacific", "Indian", "Southern", "Arctic"];

    let mut builder = tabled::builder::Builder::default().set_header(["#", "Ocean"]);
    for (i, ocean) in oceans.iter().enumerate() {
        builder = builder.add_row([i.to_string(), ocean.to_string()]);
    }

    let table = builder
        .build()
        .with(Header(message))
        .with(Footer(link))
        .with(Modify::new(Cell(0, 0)).with(MaxWidth::wrapping(5)))
        .with(Style::GITHUB_MARKDOWN);

    println!("{}", table);
}

```

```
|The t         |
|erms          |
|"the          |
|--------------+
|      #       |       Ocean        |
```

**Comment by zhiburt** [comment]
The issue is actually not in maxWidth
But when 2 panes are used with different length

**Comment by zhiburt** [comment]
```rust

#[test]
fn render_row_span_with_different_length() {
    let mut grid = Grid::new(3, 2);
    grid.set_cell_borders(DEFAULT_CELL_STYLE.clone());

    grid.set(
        &Entity::Cell(0, 0),
        Settings::new().text("first row").span(2),
    );
    grid.set(&Entity::Cell(1, 0), Settings::new().text("0"));
    grid.set(&Entity::Cell(1, 1), Settings::new().text("1"));
    grid.set(&Entity::Cell(2, 0), Settings::new().text("a longer second row").span(2));

    let expected = concat!(
        "+-------------------+\n",
        "|first row          |\n",
        "+-------------------+\n",
        "|0        |1        |\n",
        "+---------+---------+\n",
        "|a longer second row|\n",
        "+-------------------+\n",
    );

    println!("{}", grid);

    assert_eq!(expected, grid.to_string());
}
```

**Comment by zhiburt** [comment]
closed in 06a4522f21d32a8cd293c57a2d7eacc612c8f1e4
